src/main/java/racinggame/view/ResultView.java

```diff
@@ -18,15 +27,43 @@ public static void resultSign() {
         out.println(RESULT_SIGN);
     }
 
-    public static void report(List<MovingDistance> report) {
-        for (MovingDistance movingDistance : report) {
-            out.println(createResult(movingDistance));
+    public static void report(Cars cars) {
+        for (Car car : cars.getAll()) {
+            out.println(createResult(car));
         }
         br();
     }
 
-    private static String createResult(MovingDistance movingDistance) {
-        return DASH.repeat(Math.max(0, movingDistance.distance()));
+    private static String createResult(Car car) {
+        return car.toStringName() + COLON + DASH.repeat(
+            Math.max(0, car.movingDistance().distance()));
```

Comment:
![스크린샷 2023-11-05 오후 8 37 35](https://github.com/next-step/java-racingcar/assets/12059592/7490f89a-5038-4a6c-8ff2-4fe1f3b89594)

```suggestion
        return car.toStringName() + COLON + DASH.repeat(car.totalDistance());
```

`한줄에 점하나` 어떠한 의미일까요?

Reply:
해당 점이 여러개가 되면 해당하는 객체들간의 의존도가 강해지기 때문인듯 합니다.
평소 문자열에 대해서는 해당 생각을 해본적이 없는데 의견 주셔서 감사합니다 : )